xmtp_mls/src/api/test_utils.rs

```diff
@@ -176,3 +181,34 @@ mod wasm {
         }
     }
 }
+
+pub async fn wait_for_some<F, T>(f: F) -> Option<T>
+where
+    F: Fn() -> Option<T>,
+{
+    let start = Instant::now();
```

Comment:
Try to prefer https://docs.rs/web-time/latest/web_time/ `Instant` fn, so that it works in webassembly too

Reply:
#1366 puts all these into a common crate for easier access, but that PR still needs some love to get tests passing

Reply:
Cool, thanks